vue/components/ui/generic-field-platforme/generic-field-platforme.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div class="generic-field-container">
+        <img class="icon" v-bind:src="iconSrc" v-if="iconSrc" v-show="iconVisible" />
+        <input
+            class="generic-field-platforme"
+            v-bind:class="[iconVisible ? '' : 'icon-invisible', disabled ? 'disabled' : '']"

```

Comment:
I think you can write `{icon-visible: iconVisible}` or something similar for a shorter form (see https://vuejs.org/v2/guide/class-and-style.html).